views/components/ChartList.jsx

```diff
@@ -45,54 +48,78 @@ const ChartList = () => {
       console.error(error, '*****')
     }
   }
-
+  const handleShareChart = (chart) => {
+    setChartState(chart)
+  }
   return (
-    <Table>
-      <TableHead>
-        <TableRow>
-          <TableCell align='center'>Title</TableCell>
-          <TableCell align='center'>Description</TableCell>
-          <TableCell align='center'>Duplicate</TableCell>
-          <TableCell align='center'>Edit</TableCell>
-          <TableCell align='center'>Delete</TableCell>
-        </TableRow>
-      </TableHead>
-      <TableBody>
-        {chartList.map(({ title, description, _id }) => (
-          <TableRow key={_id}>
-            <TableCell align='center'>
-              <Link color='textPrimary' href={routes.chart(_id)}>
-                {title?.toUpperCase()}
-              </Link>
-            </TableCell>
-            <TableCell align='center'>{description?.toUpperCase()}</TableCell>
-            <TableCell align='center'>
-              <Button
-                type='button'
-                variant='text'
-                onClick={() => onDuplicateChart(_id)}
-              >
-                <PlaylistAdd />
-              </Button>
-            </TableCell>
-            <TableCell align='center'>
-              <Link href={routes.editChart(_id)} color='textPrimary'>
-                <Edit />
-              </Link>
-            </TableCell>
-            <TableCell align='center'>
-              <Button
-                type='button'
-                variant='text'
-                onClick={() => removeChart(_id)}
-              >
-                <Delete />
-              </Button>
-            </TableCell>
+    <>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell align='center'>Title</TableCell>
+            <TableCell align='center'>Description</TableCell>
+            <TableCell align='center'>Duplicate</TableCell>
+            <TableCell align='center'>Edit</TableCell>
+            <TableCell align='center'>Share</TableCell>
+            <TableCell align='center'>Delete</TableCell>
           </TableRow>
-        ))}
-      </TableBody>
-    </Table>
+        </TableHead>
+        <TableBody>
+          {chartList.map((chart) => (
+            <TableRow key={chart._id}>
+              <TableCell align='center'>
+                <Link color='textPrimary' href={routes.chart(chart._id)}>
+                  {chart?.title?.toUpperCase()}
```

Comment:
when would we not have a `chart`? I don't think we need the `?` operator in these cases.

```suggestion
                  {chart.title?.toUpperCase()}
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/131/commits/4b7a005d45471fc07236b428f61a2f58b4284d6a